<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqSMTKInfoPanel</class>
 <widget class="QWidget" name="pqSMTKInfoPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>1071</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Selection">
      <attribute name="title">
       <string>Selection</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>130</height>
          </size>
         </property>
         <property name="title">
          <string>Mesh Information</string>
         </property>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>15</x>
            <y>33</y>
            <width>35</width>
            <height>16</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Type:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="entityType">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>33</y>
            <width>123</width>
            <height>16</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(0, 0, 0, 0);border: none;</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>15</x>
            <y>60</y>
            <width>102</width>
            <height>16</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Number of Cells:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="entityNumberOfCells">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>60</y>
            <width>123</width>
            <height>16</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(0, 0, 0, 0);border: none;</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_16">
          <property name="geometry">
           <rect>
            <x>15</x>
            <y>90</y>
            <width>110</width>
            <height>16</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Number of Points:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="entityNumberOfPoints">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>90</y>
            <width>123</width>
            <height>16</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(0, 0, 0, 0);border: none;</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <zorder>entityType</zorder>
         <zorder>label_15</zorder>
         <zorder>entityNumberOfCells</zorder>
         <zorder>label_16</zorder>
         <zorder>entityNumberOfPoints</zorder>
         <zorder>label_14</zorder>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>888</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Visualization">
      <attribute name="title">
       <string>Visualization</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="properties">
         <property name="title">
          <string>Properties</string>
         </property>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <property name="spacing">
           <number>4</number>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Path:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="filename">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 0, 0, 0);border: none;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="filenameLabel">
            <property name="text">
             <string>Filename:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="path">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 0, 0, 0);border: none;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="pqTreeWidget" name="compositeTree">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <column>
          <property name="text">
           <string>Data Hierarchy</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Information</string>
         </property>
         <layout class="QGridLayout">
          <item row="2" column="0" colspan="2">
           <widget class="QStackedWidget" name="dataTypeProperties">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="DataSet">
             <layout class="QGridLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="1" column="1">
               <widget class="QLineEdit" name="numberOfPoints">
                <property name="cursor">
                 <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="acceptDrops">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgba(0, 0, 0, 0);border: none;</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Number of Cells:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="numberOfCells">
                <property name="cursor">
                 <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="acceptDrops">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgba(0, 0, 0, 0);border: none;</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Number of Points:</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="Table">
             <layout class="QGridLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Number of Rows:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="numberOfRows">
                <property name="cursor">
                 <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="acceptDrops">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgba(0, 0, 0, 0);border: none;</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Number of Columns</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="numberOfColumns">
                <property name="cursor">
                 <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="acceptDrops">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgba(0, 0, 0, 0);border: none;</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="memory">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 0, 0, 0);border: none;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Type:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="type">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 0, 0, 0);border: none;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Memory:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Data Arrays</string>
         </property>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <property name="spacing">
           <number>4</number>
          </property>
          <item row="0" column="0">
           <widget class="QLineEdit" name="dataTimeLabel">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 0, 0, 0);border: none;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="pqTreeWidget" name="dataArrays">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Data Type</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Data Ranges</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupExtent">
         <property name="title">
          <string>Extents</string>
         </property>
         <layout class="QGridLayout">
          <item row="2" column="1">
           <widget class="QLineEdit" name="zExtent">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 0, 0, 0);border: none;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="yExtent">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 0, 0, 0);border: none;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="xExtent">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 0, 0, 0);border: none;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Z Extent</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Y Extent:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>X Extent:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupDataTime">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Time</string>
         </property>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <property name="spacing">
           <number>4</number>
          </property>
          <item row="0" column="0">
           <widget class="pqTreeWidget" name="timeValues">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <column>
             <property name="text">
              <string>Index</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Value</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Bounds</string>
         </property>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <property name="spacing">
           <number>4</number>
          </property>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Z range:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="zRange">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 0, 0, 0);border: none;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="yRange">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 0, 0, 0);border: none;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>X range:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="xRange">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 0, 0, 0);border: none;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Y range:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
      <zorder>properties</zorder>
      <zorder>groupBox</zorder>
      <zorder>compositeTree</zorder>
      <zorder>groupBox_2</zorder>
      <zorder>groupExtent</zorder>
      <zorder>groupBox_3</zorder>
      <zorder>groupDataTime</zorder>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>pqTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
